config/routes/ezplatform_date_based_publisher.yaml

```diff
@@ -1,3 +1,6 @@
 ezplatform.ee.date_based_publisher:
     resource: '@EzSystemsDateBasedPublisherBundle/Resources/config/routing.yaml'
+
+ezplatform.ee.date_based_publisher_rest:
+    resource: '@EzSystemsDateBasedPublisherBundle/Resources/config/routing_rest.yaml'
```

Comment:
Doesn't this change the behavior of `ezplatform.ee.date_based_publisher` route? `prefix` is gone from there now.

Reply:
No, the prefix is defined below for this route file

Reply:
@alongosz Prior to this change, DBP had no non-REST routes AFAIK.